scripts/kconfig/kconfigfunctions.py

```diff
@@ -5,6 +5,19 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import os
+import sys
+
+ZEPHYR_BASE = os.environ.get("ZEPHYR_BASE")
+sys.path.insert(0, os.path.join(ZEPHYR_BASE, "scripts/dts"))
+
+import edtlib
+
+DTS_POST_CPP = os.environ["DTS_POST_CPP"]
+DTS_ROOT_BINDINGS = os.environ.get("DTS_ROOT_BINDINGS")
```

Comment:
`BINDINGS_DIR` might be better (was thinking "what's a root binding, and how is it different from a regular binding?")

Might be nice to have a comment that explains what it means when it's unset, and when it would happen.